frontend/apps/web/src/index.html

```diff
@@ -16,7 +16,7 @@
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
     <link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png" />
-    <link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png" />
+    <link rel="icon" type="image/x-icon" href="favicon.ico" />
```

Comment:
search for favicon-16x16 across the app, there could be another usage of this file, maybe in site.webmanifest?

Reply:
![image](https://user-images.githubusercontent.com/85948457/144631956-e4104cc6-6214-4906-8200-cf83c6118da3.png)